### DISCUSSION TITLE
How to increase image clarity?

### DISCUSSION THREAD

**Discussion by arahasya** [ImageMagick/ImageMagick#1907] [General]
I am displaying an image on a RGB LED Matrix 32x32, generated using Pango and this command:

sudo convert -background black -fill white -gravity center -size 32x32 \
	-pointsize 16 pango:"एस" label32.png

![WhatsApp Image 2020-04-19 at 00 19 11](https://user-images.githubusercontent.com/59508143/79667070-86ed4280-81d3-11ea-880f-eeb6f45945a2.jpeg)

The image is displayed almost perfect but can see some noise. Are there any more tricks to improve the output image? Or to fit a particular resolution?
I am using Noto-Sans font

**Comment by fmw42** [comment]
Where in this image is noise? Is the noise coming from the font and label: or from your background image. Your command is not a complete command, since it does not show how you put the text image onto your background image.

**Reply by arahasya** [reply]
Sorry for not explaining properly, By noise I meant the 3rd set of pixels around the letters aren't as bright as the main line. All the pixels should be bright equally for it to be a perfect image. Now maybe I'm seeing this because it is a very small resolution and won't be visible as I make bigger images?

For comparison this is the output with Unicode data:

![WhatsApp Image 2020-04-19 at 00 35 07](https://user-images.githubusercontent.com/59508143/79668972-0714a780-81d6-11ea-9702-98229745d8db.jpeg)


As you can see all the pixels have uniform brightness. I'm having a hard time displaying other languages so I am using images to display

**Reply by fmw42** [reply]
I am not an expert, but that might just be the way the font glyphs are designed.

But try adding +antialias to your commands to disable anti-aliasing of the text.
